setup.py

```diff
@@ -130,7 +130,10 @@ def get_version_info():
         GIT_REVISION = "Unknown"
 
     if not ISRELEASED:
-        FULLVERSION += '.dev0+' + GIT_REVISION[:7]
+        import time
+
+        time_stamp = time.strftime("%Y%m%d%H%M%S", time.localtime())
```

Comment:
It might be good to add a comment saying this is for consistency with <some wheel upload process>

Reply:
That comment is no longer needed, this is the only version string - the additional manipulation in the `numpy-wheel` repo can simply be removed.